### ISSUE TITLE
Change Pool Array to Packed Array

### ISSUE THREAD

**Issue by NutmegStudio** [GodotNativeTools/godot-d#65]
Godot d wont build on the latest godot master.
Pool Array has been depricated, packed array is the way to go.

**Comment by sheepandshepherd** [comment]
Yes, it'll be updated for 4.0 in a few months. Currently GDNative itself doesn't work in 4.0 yet due to all the active development and breaking changes in the engine.
